packages/dbus_glib.rb

```diff
@@ -1,36 +1,23 @@
 require 'package'
 
 class Dbus_glib < Package
-  description 'The D-Bus GLib package contains GLib interfaces to the D-Bus API.'
-  homepage 'http://www.linuxfromscratch.org/blfs/view/svn/general/dbus-glib.html'
-  version '0.110'
+  description 'An obsolete, primarily unmaintained glib binding for libdbus.'
```

Comment:
Why the description change?  The old one seems to be more descriptive?

Reply:
because the wiki says its unmantained

Reply:
It's OK to mention that but maybe just append to the current description?

Reply:
This is the description from the new homepage.
```
dbus-glib is a deprecated API for use of D-Bus from GLib applications.
```